test/index.test.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const Bookshelf = require('bookshelf')(require('knex')({
+  client: 'sqlite3',
+  connection: {
+    filename: './test/test.sqlite3'
+  }
+}));
```

Comment:
In order to test the plugin, I found that I need to have a database for Bookshelf to connect to and that I needed tables for my models to connect to. Instead of having everyone initialize a local db for two empty tables, I just created it as a sqlite db and included the file in source control. Open to thoughts on better ways to handle this.